aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move

```diff
@@ -123,7 +123,7 @@ module supra_framework::pbo_delegation_pool {
     use aptos_std::smart_table::{Self, SmartTable};
     use aptos_std::fixed_point64::{Self, FixedPoint64};
 
-    use supra_framework::coin::Coin;
+    use supra_framework::coin::{Coin, balance};
```

Comment:
Avoid direct import of `balance`, I believe you will still be able to call `coin::balance`, if required you can use `coin::{Self,Coin}`

Reply:
Okay, updated.